example/morphology.cpp

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright 2021 Prathamesh Tagore <prathameshtagore@gmail.com>
+//
+// Use, modification and distribution are subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+#include <boost/gil/extension/io/png.hpp>
+#include <boost/gil/image_processing/morphology.hpp>
```

Comment:
This is just an example, why not just?
```
#include <boost/gil.hpp>
```

Reply:
When I did that, my compilers started shouting at me with messages such as 
  ```identifier "morphological_gradient" is undefined```
  The program works perfectly though.
  If that is fine, I don't mind including ```boost/gil.hpp```  instead of ```boost/gil/image_processing/morphology.hpp``` after including the latter in .hpp file of former .

Reply:
Every public header has to be added to `boost/gil.hpp`, so please add yours too.